swift-sdk/Internal/RequestCreator.swift

```diff
@@ -466,7 +466,7 @@ struct RequestCreator {
         var body = [AnyHashable: Any]()
         setCurrentUser(inDict: &body)
         
-        if let buttonIdentifier, !buttonIdentifier.isEmpty {
+        if let buttonIdentifier = buttonIdentifier, !buttonIdentifier.isEmpty {
             body.setValue(for: JsonKey.embeddedButtonId, value: buttonIdentifier)
```

Comment:
Wow didn't know how the initial one just worked like that! 
New one is clearer 👍🏼